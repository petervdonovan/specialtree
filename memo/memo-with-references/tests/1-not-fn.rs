use memo_with_references::memo;

#[memo]
enum Huh {}

fn main() {}
