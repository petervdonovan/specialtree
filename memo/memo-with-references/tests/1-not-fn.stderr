error: expected fn item
 --> tests/1-not-fn.rs:4:1
  |
4 | enum Huh {}
  | ^^^^^^^^^^^
